apps/src/feedback.js

```diff
@@ -158,6 +158,8 @@ FeedbackUtils.prototype.displayFeedback = function(options, requiredBlocks,
   var finalLevel = (options.response &&
     (options.response.message === "no more levels"));
 
+  var userId = (options.response && options.response.user_id);
```

Comment:
Am I wrong in thinking that the parentheses are superfluous? Can they be removed?